doc/README

```diff
@@ -76,6 +76,9 @@ The following environment variables affect *python-afl* behavior:
    *py-afl-fuzz* sets this variable automatically,
    so there should normally no need to set it manually.
 
+ ``PYTHON_AFL_TSTL``
+   TSTL test harness code is ignored only if this variable is set.
```

Comment:
I don't like the way it's phrased. It seems to imply that this variable should normally be set, whereas most users (who don't use TSTL) won't need it.
How about this?
> if this variable is set, TSTL test harness code is ignored.

Bonus points for making "TSTL" hyperlink to the project's homepage.